python/pyarrow/table.pxi

```diff
@@ -357,6 +509,29 @@ cdef class ChunkedArray(_PandasConvertible):
         Returns
         -------
         cast : Array or ChunkedArray
+
+        Examples
+        --------
+        >>> import pyarrow as pa
+        >>> n_legs = pa.chunked_array([[2, 2, 4], [4, 5, 100]])
+        >>> n_legs.type
+        DataType(int64)
+        >>> n_legs.cast(pa.duration('s')).type
+        DurationType(duration[s])
+        >>> n_legs.cast(pa.duration('s'))
+        <pyarrow.lib.ChunkedArray object at ...>
+        [
+          [
+            2,
+            2,
+            4
+          ],
+          [
+            4,
+            5,
+            100
+          ]
+        ]
```

Comment:
This is a bit unclear.
I think it might improve if we assign the output of `cast` to some variable and show that the type of the variable changed. Like
```
>>> n_legs_seconds = n_lefts.cast(pa.duration("s"))
>>> n_legs_seconds.type
DurationType(duration[s])
```

Reply:
Sure, can do. I will also divide the example with text so it will be clearer. What I can also do is remove the last call `>>> n_legs.cast(pa.duration('s'))` altogether.